homer0/parserror
# Replace Travis with GitHub actions

### What does this PR do?

[Update with the reasons why I'm migration].

### How should it be tested manually?

Check the PR checks to see the action working, and, of course...

```bash
yarn test
# or
npm test
```
